README.md

```diff
@@ -9,11 +9,14 @@ Automatically dump and archive PostgreSQL backups to Amazon S3
 ## Setup
 
 Add secrets.conf:
-  - PG_HOST and PG_USER with your PostgreSQL hosts and backup user.
-  - PG_DB with pg connection string
-  - S3_PATH with your Amazon S3 bucket and path
-  - AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY with your aws credentials
-  - MAX_DAYS number of days before backups deleted
+  - PG_HOST - PostgresSQL host
+  - PG_USER - PostgreSQL backup user
```

Comment:
Where are PG_HOST and PG_USER used?

Reply:
They actually aren't used from what I'm seeing